smartcar/smartcar.py

```diff
@@ -85,7 +85,7 @@ def __init__(self, client_id, client_secret, redirect_uri, scope=None, test_mode
         else:
             self.test_mode = test_mode if test_mode else False
 
-    def get_auth_url(self, force=False, state=None, vehicle_info=None):
+    def get_auth_url(self, force=False, state=None, vehicle_info=None, single_select=None):
```

Comment:
The documentation above says single_select defaults to False, so might want to change None --> False! However, I don't think it really matters right now because you're only adding single_select if it is True (and omission is false by default haha)

Reply:
Good catch! Fixed